docs/execution-guide.md

```diff
@@ -32,43 +31,78 @@ PID|1|7777555^4^M11^test^MR^University Hospital^19241011^19241012|PATID7755^5^M1
+2. Running data through the pipeline
+3. Viewing data in the FHIR server
+
+### Access your Azure account 
+> **Note**: If this is your first time running data through the pipeline, we recommend having this guide and the Azure portal open side-by-side.  
+
+1. Open [https://portal.azure.com/](https://portal.azure.com/) in your browser and log in with your username and password.![azure-portal](./images/azure-portal.png)
+1. Make sure you're logged into the account that has access to the Azure resource group you have used so far. To check, go to the "Azure services" heading and click on `Resource groups`.![azure-portal-resource-groups](./images/azure-portal-resource-groups.png) 
+1. Click into the appropriate Azure resource group.![azure-portal-resource-group](./images/azure-portal-resource-group.png) 
+
+### Upload and run data through the pipeline
+> **Note**: If you prefer, you can upload data using the [Azure Storage Explorer Tool](https://azure.microsoft.com/en-us/products/storage/storage-explorer/). We don't provide instructions for using that tool here, but the broad strokes will be the same - you'll need to upload `sample-data/VXU-V04-01_success_single.hl7` to the `source-data` container in your PHI storage account.
+
+1. Within your `Resource group`, filter down to view only `Storage account` type resources. To do so, click into the `Filter for any field...` search box and type in `Storage account`.![azure-filter-storage-accounts](./images/azure-filter-storage-accounts.png)
+1. Click into the PHI storage account, which is where all Protected Health Information is stored outside of the FHIR server. The precise name of the storage bucket will have the form `phdi{environment}phi{clientId}`, e.g., `phdidevphi1667849158`.![azure-select-phi-bucket](./images/azure-select-phi-bucket.png)
+1. Go to the left sidebar and under the `Data storage` header, click `Containers`. ![azure-containers](./images/azure-containers.png)
+1. Click into the `source-data` container.![azure-select-source-data-container](./images/azure-select-source-data-container.png)
+1. Then click into the `vxu` folder.![azure-select-vxu-folder](./images/azure-select-vxu-folder.png)
+1. Now we're ready to run a VXU message through the pipeline! First, click the `Upload` button in the toolbar.![azure-upload](./images/azure-upload.png)
+1. Then click `Browse for files`.![azure-browse](./images/azure-browse.png)     
+1. Navigate to the folder on your computer where you've downloaded or forked the `sample-data/` folder and select the `VXU-V04-01_success_single.hl7` file. Click `Open`, then click `Upload` to upload this file into the `source-data/vxu/` directory of your PHI bucket.![azure-upload-file](./images/azure-upload-file.png)    
+
+     > **Note**: because the ingestion pipeline is event-driven, simply uploading the file is all that is required to trigger the pipeline. There is an event listener monitoring the PHI bucket for file creation events.
```

Comment:
It would be great to link to our debugging guide for the ReadSourceData function app here - I'm not sure if it's quite ready yet, but I believe @DanielPaseltiner was working on it!

Reply:
I added this to the troubleshooting section at the bottom of the execution guide (since the first how to section is intended for running the success sample VXU message). Also added a bunch of collapsible sections so users can jump to troubleshooting--let me know what you think!